nintendo/nex/prudp.py

```diff
@@ -886,6 +928,8 @@ def send_packet(self, packet):
 			self.ack_events[packet.packet_id] = event
 		
 		self.send_packet_raw(packet)
+		if packet.type == TYPE_PING:
+			self.send_packet_raw(packet)
```

Comment:
This was sort of just for testing, when observing I saw that sometimes PING packets are sent multiple times, which shouldn't break anything (hence the `self.packet_id_in_ping = packet.packet_id + 1` instead of `self.packet_id_in_ping += 1`)